openrtb_ext/convert_down.go

```diff
@@ -31,13 +31,6 @@ func ConvertDownTo25(r *RequestWrapper) error {
 		}
 	}
 
-	// Remove fields introduced in OpenRTB 2.6+. The previous OpenRTB 2.5 spec did not specify that
-	// bidders must tolerate new or unexpected fields.
-	clear26Fields(r)
```

Comment:
I thought we just want to drop the `Imp` fields?

Reply:
See https://github.com/prebid/prebid-server/issues/3613.

I think we want to delete these function calls, but not the functions themselves so adapters can use them, so that all 2.6 fields are passed through by default. Specific ext fields that are called out on that issue that need to be moved should be cleared when down converting (`regs.ext.gdpr/us_privacy`, source.ext.schain`, `user.ext.eids/consent`). Note that `imp.prebid.is_rewarded_inventory` should be copied and not moved, in which case both it and `imp.rwdd` should be passed through without clearing.
I suggest we merge this PR and then I can put up another PR that adds a function that only clears the moved ext fields during a down convert.